releasenotes/notes/min-reached-finder-flag-aa6dd9021e165f80.yaml

```
@@ -0,0 +1,10 @@
+---
+features:
+  - |
+    A new ``min_reached`` field has been added to the metadata outputted by :func:`circuit_knitting.cutting.find_cuts` to check if the cut-finder found
+    a cut scheme that minimized the sampling overhead. Note that the search algorithm employed by the cut-finder is `guaranteed` to find
```

Is you goal to italicize this?
```suggestion
    a cut scheme that minimized the sampling overhead. Note that the search algorithm employed by the cut-finder is *guaranteed* to find
```